demo/standalone.php

```diff
@@ -0,0 +1,371 @@
+<?php
```

Comment:
I guess this is a copy/paste of demo script that uses different autoloader.

Please instead add commented-out fragment using different autoloader in original demo file.

Also please run `wiki-doc.php` file to regenerate documentation page using that file.